content/blog/release-1-8-0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Rivet 1.8.0 release
+description: Rivet 1.8.0 is now available. This release adds border removal utility classes and fixes bugs related to the button, dropdown, and collapsible components.
+date: "2021-02-17"
```

Comment:
We can go ahead and merge this into develop, but we'll probably want to update the date when we make this post live (aka merge into master). We'll need to do the release/publish to npm before we make this live so that might be a couple of days before we can publish.